stats_test.go

```diff
@@ -335,6 +377,8 @@ func TestExporter_makeReq(t *testing.T) {
 				return
 			}
 			if !reflect.DeepEqual(resps, tt.want) {
+				temp := reflect.DeepEqual(resps, tt.want)
```

Comment:
was this accidentally left in?

Reply:
Yes.  This code is for debugging.  I would remove it.